pkg/token/daemon.go

```diff
@@ -317,3 +265,74 @@ func Tokend(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (error, <
 
 	return nil, shutdownChan
 }
+
+// GetToken makes ZTS API calls to generate an X.509 role certificate
+func GetToken(cfg *config.IdentityConfig, certPEM, keyPEM []byte) (roletokens [](*RoleToken), accesstokens [](*AccessToken), err error) {
```

Comment:
I know `GetToken()` is simply copied but if it is not used outside of the code, it could be set as a private function `getToken()`